### DISCUSSION TITLE
Can't install gnuradio on Macbook M1

### DISCUSSION THREAD

**Discussion by Vinz87** [Homebrew/discussions#1092] [Everyday usage]
I tried `brew install gnuradio`, and I get this error:
```
Error: gnuradio: no bottle available!
You can try to install from source with:
  brew install --build-from-source gnuradio
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

I tried the suggested `--build-from-source` option, but then I get the same "no bottle available" error for _volk_.

Is there any plan to make gnuradio compatible with Apple Silicon? (Macbook Pro M1, macOS 11.2.3)
Thank you

**Comment by SMillerDev** [comment]
> Is there any plan to make gnuradio compatible with Apple Silicon? (Macbook Pro M1, macOS 11.2.3)

You'll have to ask the upstream developers. Homebrew only ships software. We don't create or support it.
